server/db.json

```diff
@@ -0,0 +1,204 @@
+      "type": "recycle",
+      "imageSrc": "/recycle_07.svg",
+      "score": 1,
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": 1208,
+      "description": "used foodbox",
+      "type": "recycle",
+      "imageSrc": "/recycle_08.svg",
+      "score": 1,
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": 1209,
+      "description": "used smoothie cup",
+      "type": "recycle",
+      "imageSrc": "/recycle_09.svg",
+      "score": 1,
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": 1210,
+      "description": "compressed plastic bottle",
+      "type": "recycle",
+      "imageSrc": "/recycle_10.svg",
+      "score": 1,
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": 1211,
+      "description": "green plastic bag",
+      "type": "recycle",
+      "imageSrc": "/recycle_11.svg",
+      "score": 1,
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": 1212,
+      "description": "white plastic bag",
+      "type": "recycle",
+      "imageSrc": "/recycle_12.svg",
+      "score": 1,
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": 1101,
+      "description": "old magazine",
+      "type": "paper",
+      "imageSrc": "/paper_01.svg",
+      "score": 1,
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": 1102,
+      "description": "old letters",
+      "type": "paper",
+      "imageSrc": "/paper_02.svg",
+      "score": 1,
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": 1103,
+      "description": "old package",
+      "type": "paper",
+      "imageSrc": "/paper_03.svg",
+      "score": 1,
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": 1104,
+      "description": "old brown letter",
+      "type": "paper",
+      "imageSrc": "/paper_04.svg",
+      "score": 1,
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": 1001,
+      "description": "banana peel",
+      "type": "residual",
+      "imageSrc": "/bio_01.svg",
+      "score": 1,
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": 1002,
+      "description": "eggshell",
+      "type": "residual",
+      "imageSrc": "/bio_02.svg",
+      "score": 1,
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": 1003,
+      "description": "lettuce leaf leftover",
+      "type": "residual",
+      "imageSrc": "/bio_03.svg",
+      "score": 1,
+      "credits": "Lebensmittel Vektor erstellt von brgfx - de.freepik.com"
+    },
+    {
+      "id": 1004,
```

Comment:
peppers, lettuce, eggshell, banana peel, fries, pizza, bone... and so on is biodegradable and therefore should be the type "bio"

Reply:
thx, this will change in level 2 :-)